### DISCUSSION TITLE
Extracting outside layer of particles

### DISCUSSION THREAD

**Discussion by Nadun19** [Xiangyu-Hu/SPHinXsys#493] [Q&A] [answered]
Hello,

I am following  `test_3d_particle_relaxation_single_resolution` example to generate particles from an stl file. After particle relaxation I would like to extract just the outermost layer of particles to be turned into a solidboundary. 

Thanks

**Comment by Xiangyu-Hu** [comment, marked as answer]
I guess that you can simply use level set threshold to peel the first layer particle off.

**Reply by Nadun19** [reply]
Thank you, this approach worked!

**Reply by Xiangyu-Hu** [reply]
You are welcomed to merge this feature to the master by pull request.
